shop/src/main/resources/db/migration/V1_1_0__create_location_schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE Location (
+     id UUID NOT NULL,
+     name VARCHAR(30),
+     address_country VARCHAR(30),
+     address_city VARCHAR(30),
+     address_county VARCHAR(30),
+     address_streetaddress VARCHAR(30),
+     PRIMARY KEY (id)
+);
```

Comment:
You do not need to change anything, but it's worth mentioning that one migration would be enough to create the initial database structure. It can contain all the `CREATE TABLE` statements as it is easier to see all the tables that have been created in the first "iteration" of the project  in one file. 

But using multiple migrations as you did is not wrong and for sure it helped you understand how Flyway works.